apps/session-manager/src/app.ts

```diff
@@ -8,17 +8,23 @@ import {
   NodeEnvironmentEnum,
   getNodeEnvironmentFromProcessEnv,
 } from "@pagopa/ts-commons/lib/environment";
+import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
 import bearerSessionTokenStrategy from "./auth/session-token-strategy";
 import { RedisRepo, FnAppRepo } from "./repositories";
 import { attachTrackingData } from "./utils/appinsights";
-import { getRequiredENVVar } from "./utils/environment";
+import { getENVVarWithDefault, getRequiredENVVar } from "./utils/environment";
 import { SessionController } from "./controllers";
 import { httpOrHttpsApiFetch } from "./utils/fetch";
 import { toExpressHandler } from "./utils/express";
 import { withUserFromRequest } from "./utils/user";
 import { getFnFastLoginAPIClient } from "./repositories/fast-login-api";
-import { generateNonceEndpoint } from "./controllers/fast-login";
 import { getLollipopApiClient } from "./repositories/lollipop-api";
+import {
+  fastLoginEndpoint,
+  generateNonceEndpoint,
+} from "./controllers/fast-login";
```

Comment:
Import the controllers from the index file, ex `FastLoginController`.

Reply:
done in https://github.com/pagopa/io-auth-n-identity-domain/pull/22/commits/0aac1a7af5e32060657c60e53ef458e41e4e445c